php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -461,6 +471,43 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                 case 'table':
                     $this->testName = trim($pieces[1]);
                     $this->table    = trim($pieces[1]);
+
+                    // Add things that are added by addMetaDatafields()
+                    // but aren't in the .linst file to dictionary
+                    $this->dictionary = array_merge(
+                        $this->dictionary,
+                        [
+                            new DictionaryItem(
+                                'Date_taken',
+                                'Date of Administration',
+                                $scope,
+                                new \LORIS\Data\Types\DateType(),
+                                new Cardinality(Cardinality::SINGLE),
+                            ),
+                            new DictionaryItem(
+                                'Candidate_age',
+                                'Candidate Age (Months)',
+                                $scope,
+                                new \LORIS\Data\Types\Duration(),
+                                new Cardinality(Cardinality::SINGLE),
+                            ),
+                            new DictionaryItem(
+                                'Window_Difference',
+                                'Window difference from test battery (days)',
+                                $scope,
+                                new \LORIS\Data\Types\Duration(),
+                                new Cardinality(Cardinality::SINGLE),
+                            ),
+                            new DictionaryItem(
+                                'Examiner',
+                                'Examiner',
+                                $scope,
+                                // Should this be an enum of examiners?
+                                new StringType(255),
+                                new Cardinality(Cardinality::SINGLE),
+                            ),
+                        ]
+                    );
```

Comment:
would all this above be better placed after the call to  _addMetadataFields() on line 520  like how its organized for the other fields?

Reply:
I'll look into it, I think there was originally a reason it had to be here but I can't remember what it was. (I think maybe the initial version depended on testName but that doesn't seem to be the case anymore?)

Reply:
I've moved this code into the addMetadataFields itself so that it properly handles the conditionals.